keras_microscopy/preprocessing/standardize.py

```diff
@@ -0,0 +1,71 @@
+import keras
+import numpy
+import skimage.exposure
+import skimage.restoration
+import skimage.transform
+
+
+def desaturate(ratio):
+    """
+    Fades the image to white. Higher ratio means more fading.
+    :param ratio: The ratio to fade by, between 0 and 1.
+    :return: The desaturate function.
+    """
+    return lambda x: x + (x.max() - x) * ratio
+
+
+def rescale(scale, **kwargs):
+    """
+    Rescales the image according to the scale ratio.
+    :param scale: The scalar to rescale the image by.
+    :param kwargs: Additional arguments for skimage.transform.resize.
+    :return: The rescale function.
+    """
+    if keras.backend.image_data_format() == 'channels_first':
+        return lambda x: skimage.transform.resize(x, (len(x[:, 0, 0]), len(x[0, :, 0]) * scale, len(x[0, 0, :]) * scale), **kwargs)
+    else:
+        return lambda x: skimage.transform.resize(x, (len(x[:, 0, 0]) * scale, len(x[0, :, 0]) * scale, len(x[0, 0, :])), **kwargs)
+
+
+def equalize(**kwargs):
+    """
+    Equalizes the image histogram, per channel.
+    :param kwargs: Additional arguments for skimage.exposure.equalize_hist.
+    :return: The equalize function.
+    """
+    if keras.backend.image_data_format() == 'channels_first':
+        def f(x):
+            y = numpy.empty_like(x, dtype=numpy.float64)
+            for i in range(len(x[:, 0, 0])):
+                y[i, :, :] = skimage.exposure.equalize_hist(x[i, :, :], **kwargs)
+            return y
+        return f
```

Comment:
Here's what it would looks like if we used `numpy.moveaxes`:

```python
def equalize(**kwargs):
    def f(x):
        if keras.backend.image_data_format() == 'channels_last':
            x = numpy.moveaxes(x, 0, -1)
        
        y = numpy.empty_like(x, dtype=numpy.float64)
        
        for index, img in enumerate(x):
            y[index] = skimage.exposure.equalize_hist(img, **kwargs)
        
        if keras.backend.image_data_format() == 'channels_last':
            y = numpy.moveaxes(y, -1, 0)

        return y
    return f
```

What is your preference?